python/py-xraylarch/Portfile

```diff
@@ -27,29 +27,20 @@ long_description    Larch is a open-source library and set of applications for p
 
 homepage            https://xraypy.github.io/xraylarch
 
-# Find the latest at https://pypi.org/project/xraylarch/#files
-master_sites        https://files.pythonhosted.org/packages/ff/4a/cf7f494aaa3ea1ba15adf5806cef0ac5136affc7e8f7c2048999fa4e28d9/
-distname            xraylarch-${version}-py3-none-any
+checksums           rmd160  e72c2a07705f70250ebc4aae1d84d8fed28fb57a \
+                    sha256  5bf2b6fcd9d19d3c30b0e5b3cca495019e0d208b17616ed9f7bbf6f78607484b \
+                    size    84753392
 
-checksums           rmd160  0736ee2e7d6ba40c8c55dc7111cb3c3067454fd2 \
-                    sha256  92d558df4c8302a277ead4ad597920285abf3b7e69ca9ab13873e8dc319e452a \
-                    size    21056444
-
-python.versions     37 38 39 310
+python.versions     38 39 310
 python.pep517       yes
```

Comment:
this is the default now, so the line can be removed. Also you shouldn't clear the `pep517_backend` below; so remove both lines